includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1482,16 +1482,42 @@ public function set_payment_method_title_for_order( $order, $payment_method_type
 	 * @return array Array of keyed metadata values.
 	 */
 	protected function get_metadata_from_order( $order, $payment_type ) {
+		switch ( static::class ) {
```

Comment:
Though this is functional as well, I think we're using `static` mostly in static methods or when we need an instance of the same class (`new static()`). Maybe switch to `get_class()` here?
```suggestion
		switch ( get_class( $this ) ) {
```